great_expectations/core/usage_statistics/anonymizers/anonymizer.py

```diff
@@ -41,12 +44,11 @@ def anonymize(self, string_):
 
     def anonymize_object_info(
```

Comment:
@cdkini This is odd.  The self._ge_classes is still defined in the parent class, but not used any more.  So what happens to that list of action classes?  Thanks.

Reply:
`self._ge_classes` still exists - this PR is one of two or three. The purpose of this PR is to remove `ge_classes` as an arg from this `anonymize_object_info` method, not to remove it entirely.

They're still strongly coupled with the Anonymizer as a result of other utilities like `_is_parent_class_recognized`. Due to the fact that usage stats is very easy to break, I want to do this in steps. That make sense?

Reply:
🙇